### ISSUE TITLE
Implement changes for CI/CD packages - 'Scanning multiple pages'

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#2290]
This issues was moved from the internal [E2E board #7173]( https://github.ibm.com/IBMa/E2E/issues/7173) to this external Checker board for implementation:

- [x] review the results of #7145 with @snidersd
- [x] confirm the changes needed,
- [ ] make the changes needed.
- [ ] add link to Wiki to the top-level readme's so users can find the higher level info to get started (per discussion in Showcase)

Based on first glance, Sharon's review was thorough and the observations & recommendations should be able to be quickly implemented in a single Sprint.

Mtg w/Sharon rescheduled to Friday 2 May 2025

some random notes & To do's from mtg w/Sharon:
- [ ] The screen shot for the stored scans is out-dated and needs to be updated in the user guide (see below)
- [ ] Run achecker to see flags 
   - [ ] -flag to get a path to results folder (or just from root)  
- [ ] Show how to get the flags, and output in the results  in file
> Run achecker after installation to get a list of flags with options to use when running the checker.
Run achecker archive to get a list of valid IDs for rule sets (policies)
Run npx achecker /path/to/file.txt to scan the files. (Note: the scan also runs without npx at the beginning)
Example: npx achecker --outputFormat csv file.txt
The accessibility-checker creates a results folder where all results are output as json files for each scan, as well as the outputFormat file if used.

Process steps:
- [ ] create a project
- [ ] running creates a results for each file scanned
- [ ] running creates a results summary
- [ ] yml file vs command line -flags.
- [ ] Run "npm ~~run~~ test" and the checker runs on all the pages listed in main.js and outputs the results to a new "results" folder.

**Comment by philljenkins** [comment]
Originally planned:
- The readme changes will be implemented in https://github.com/IBMa/equal-access/issues/2254
- The wiki changes will be implemented by this issue (no PR required)

**Comment by philljenkins** [comment]
During Showcase 28 May it was proposed and accepted to add a link to the top-level package ReadMe's to point to the Wiki for  getting started info and higer-level info such as [Setting up a basline](https://github.com/IBMa/equal-access/wiki#baseline-basics)
